packages/react-component-library/src/components/Timeline/TimelineToolbar.tsx

```diff
@@ -1,13 +1,34 @@
 import React, { useContext } from 'react'
-import { IconChevronRight, IconChevronLeft } from '@royalnavy/icon-library'
+import {
+  IconChevronRight,
+  IconChevronLeft,
+  IconZoomIn,
+  IconZoomOut,
+} from '@royalnavy/icon-library'
 
 import { Button, BUTTON_SIZE, BUTTON_VARIANT } from '../Button'
 import { StyledToolbar } from './partials/StyledToolbar'
-import { TIMELINE_ACTIONS } from './context/types'
+import { TIMELINE_ACTIONS, TimelineScaleOptions } from './context/types'
 import { TimelineContext } from './context'
 import { StyledToolbarButtons } from './partials/StyledToolbarButtons'
+import { StyledToolbarSeparator } from './partials/StyledToolbarSeparator'
+import { useTimelineScale } from './hooks/useTimelineScale'
 
-export const TimelineToolbar: React.FC = () => {
+interface TimelineToolbarProps {
+  scaleOptions: TimelineScaleOptions[]
+}
+
+export const TimelineToolbar: React.FC<TimelineToolbarProps> = ({
+  scaleOptions,
+}) => {
+  const {
+    canZoomIn,
+    canZoomOut,
+    moveNext,
+    movePrevious,
+    zoomIn,
+    zoomOut,
+  } = useTimelineScale(scaleOptions)
```

Comment:
Does this hook (`useTimelineScale`) mean the raw `GET_NEXT` and `GET_PREV` reducer actions should no longer ever be dispatched directly.

I note that timeline scaling isn't an 'optional' feature it comes as standard.

Do we maybe want to hide it behind a flag prop or use a compound component?